aws_lambda_builders/workflows/go_modules/builder.py

```diff
@@ -44,7 +46,11 @@ def build(self, source_dir_path, output_path):
         env.update(self.osutils.environ)
         env.update({"GOOS": "linux", "GOARCH": "amd64"})
         runtime_path = self.binaries[self.LANGUAGE].binary_path
-        cmd = [runtime_path, "build", "-o", output_path, source_dir_path]
+        cmd = [runtime_path, "build"]
+        if self.mode and self.mode.lower() == BuildMode.DEBUG:
```

Comment:
Are we sure that BuildMode.DEBUG is always lower? I would suggest to make it BuildMode.DEBUG.lower() as well.

Reply:
Following a pattern from the .NET builder, so the `aws_lambda_builders/workflow.py::BuildMode` enum values are contracted to be lower case, but input is not.